runtime-modules/content/src/lib.rs

```diff
@@ -1119,12 +1147,24 @@ decl_module! {
             // ensure video can be removed
             Self::ensure_video_can_be_removed(&video)?;
 
+            if !assets_to_remove.is_empty() {
```

Comment:
Is it ok to delete a video without having the data objects to remove? Is it a correct use-case?

Reply:
If I can add a video with optional assets then I can also delete it even if it has no assets